### PR TITLE
Feature/Extend Readme_dev for updating the runtime

### PR DESCRIPTION
**Pull request by Milena-Czierlinski**

### CODE REVIEW

File: README_dev.md [hunk 1]
```diff
@@ -16,3 +16,22 @@ Run `melos bootstrap` to [install and link dependencies](https://melos.invertase
...
+
+## Update the Runtime
+
+Basically, the whole enmeshed Runtime is written in the file `packages/enmeshed_runtime_bridge/assets/index.js`.
+Its respective version is specified in `packages/enmeshed_runtime_bridge/natives/package.json`.
+In order to update it, you must adjust the version there, navigate to `packages/enmeshed_runtime_bridge/natives` and run `npm i`.
+Afterwards, run `npm run build` in the same location, which will update the `index.js` file.
+
+Now, to make the changes accessible for the App, all adjustments that are exposed, e.g. DTOs, DVOs, use cases and events, need to be adjusted in their respective places.
+To do so, step through every Runtime change that was performed from the previous version to your newly updated one and check if it needs to be tranferred to the App.
```
Comment (Siolto) [inline README_dev.md:28]:
```suggestion
To do so, step through every Runtime change that was performed from the previous version to your newly updated once and check if it needs to be transferred to the App.
```

File: README_dev.md [hunk 2]
```diff
@@ -16,3 +16,22 @@ Run `melos bootstrap` to [install and link dependencies](https://melos.invertase
...
+Usually, only changes to the Content- and Runtime-library are relevant for the App, but if you aren't sure, simply use the search function and check if the respecive data object occurs anywhere besides the `index.js` file in the App-repository.
```
Comment (Siolto) [inline README_dev.md:29]:
```suggestion
Usually, only changes to the Content- and Runtime-library are relevant for the App, but if you aren't sure, simply use the search function and check if the respective data object occurs anywhere besides the `index.js` file in the App-repository.
```

**Review by Siolto — CHANGES_REQUESTED** [review]

File: README_dev.md (hunk 1 above)
Comment (Milena-Czierlinski) [reply README_dev.md:28]: I mean the newly updated version - the newly updated one.

**Review by Siolto — APPROVED** [review]
LGTM 👍

**Pull request merged**